plugins/tiff/src/tiffpresentation.cc

```diff
@@ -391,12 +402,12 @@ void TiffPresentation::done()
 }
 
 /**
- * Overriden operator+= to add two values of the same key.
+ * Add two pipette color map values of the same key.
  * 
  * Loops through rhs, adds values of the same key to each other and assigns it back to lhs.
  * Return lhs at the end.
  */
-PipetteLayerOperations::PipetteColor operator+=(PipetteLayerOperations::PipetteColor& lhs, const PipetteLayerOperations::PipetteColor& rhs)
+PipetteLayerOperations::PipetteColor sumPipetteColors(PipetteLayerOperations::PipetteColor& lhs, const PipetteLayerOperations::PipetteColor& rhs)
```

Comment:
You are taking `lhs` by reference, and then returning a non-reference.

I recommend making `lhs` a const reference. Callers may not expect you to modify it.